proposals/lip-0048.md

```diff
@@ -50,7 +50,7 @@ We define the following constants:
 | `LENGTH_TOKEN_ID` | uint32 | 8 | The length of a token ID in bytes. |
 | **Configurable Constants** | | **Mainchain Value** | |
 | `MIN_FEE_PER_BYTE` | uint64 | 1000 | Minimum amount of fee per byte required for transaction validity. |
-| `TOKEN_ID_FEE` | bytes | `TOKEN_ID_LSK = 0x0000000000000000` | Token ID of the token used to pay the transaction fees. |
+| `TOKEN_ID_FEE` | bytes | configurable | Token ID of the token used to pay the transaction fees. |
```

Comment:
same as above, I would write the explicit mainchain value

Reply:
Changed.